src/databricks/labs/ucx/account/aggregate.py

```diff
@@ -28,6 +29,14 @@ class AssessmentObject:
     failures: list[str]
 
 
+@dataclass
+class Table(HiveMetastoreTable):
```

Comment:
maybe rename to AccountTable ?

Reply:
hmm, yes, there is no such thing as an "account table", maybe `TabeWithWorkspaceId` or `WorkspaceTable`